app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -315,23 +347,20 @@ fun Starter() {
                 .fillMaxWidth()
         )
         Text(
-            text = "1인 1닭 가능할 것 같은 사람", style = TalkbbokkiTypography.b2_bold
+            text = starter, style = TalkbbokkiTypography.b2_bold
         )
     }
 }
 
 @Composable
 fun ShareBottom(
-    onClickShareLink: () -> Unit,
-    onClickScreenShot: () -> Unit
+    onClickShareLink: () -> Unit, onClickScreenShot: () -> Unit
```

Comment:
🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)